lib/routes/dialogBox.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:flutter/material.dart';
+
+class DialogBox extends StatelessWidget {
+  const DialogBox({Key? key}) : super(key: key);
+  Widget _buildDelDialog(BuildContext context) {
+    return new AlertDialog(
+      title: const Text('This is title'),
+      actions: <Widget>[
+        new FlatButton(
+          onPressed: () {
+            Navigator.of(context).pop();
+          },
+          textColor: Theme.of(context).primaryColor,
+          child: const Icon(Icons.arrow_back),
+        ),
+        new FlatButton(
+          onPressed: () {
+            Navigator.of(context).pop();
+          },
+          textColor: Theme.of(context).primaryColor,
+          child: const Icon(
+            Icons.done,
+            color: Colors.green,
```

Comment:
Please give some description about what DialogBox does and a link to its official documentation